utils/README.md

```diff
@@ -0,0 +1,52 @@
+# APAeval utils
+
+This directory provides Dockerfiles and corresponding 'recipes' for scripting/processing tasks that are common across workflows.
+
+Contents:
+- [Available utilities](#Available-utilities)
+- [Contributing instructions](#Contributing-instructions)
+
+# Available utilities
+
+## Convert GTF to BED12 (aka 'gene model') file
+
+**Description:** Convert a [GTF](https://genome.ucsc.edu/FAQ/FAQformat.html#format4) annotation file to [BED12](https://genome.ucsc.edu/FAQ/FAQformat.html#format1) (also known as a 'gene model' file) format.
+
+**Subdirectory name**: `gtf_to_bed12`
+
+**Dockerhub link:** https://hub.docker.com/r/apaeval/gtftobed
+
+**Dockerhub url (to use with Nextflow/Snakemake)**: `docker.io/apaeval/gtftobed:1.0`
+
+**Usage (shell command to run the tool)**:
+```
+/gtfTobed.py --gtf <path to GTF file> --out_bed <name of the output BED file>
```

Comment:
Will this work if someone writes `python gtfTobed.py ...` ? Just curious and wondering how the conventions are here

Reply:
```python3 gtfTobed.py``` & ```python3.8 gtfTobed.py``` both also work (```python gtfTobed.py``` does not)